src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/reader/ClusterDetailsEventProcessorTestHelper.java

```diff
@@ -33,6 +34,11 @@ public void addNodeDetails(String nodeId, String address, boolean isMasterNode)
     nodeDetails.add(createNodeDetailsMetrics(nodeId, address, isMasterNode));
   }
 
+  public static ClusterDetailsEventProcessor.NodeDetails newNodeDetails(String nodeId, String address,
```

Comment:
should we make all the arguments `final` ?

Reply:
Done